modules/ROOT/pages/migrate-cl.adoc

```diff
@@ -0,0 +1,49 @@
+* Certain CL image types are not yet supported in FCOS. Refer to the https://getfedora.org/coreos/download/[Download page] to see a list of the available image types.
+* There are no official Vagrant cloud images for FCOS, and the team has no plans to create OCI container images for FCOS. FCOS is intended to be a host OS for running containers, rather than a host for producing containers.
+
+== Configuration changes
+The xref:fcct-config.adoc[Ignition specification] has changed in both syntax and function. As a result, consider the following when you write new FCC files:
+
+* Users and groups are defined in the xref:ign-passwd.adoc[`passwd`] node of the FCC file.
+//* CLC --> FCC changes. Need more info.
+//* coreos-metadata rename - need more info.
+//* PXE custom OEMs - need more info.
+* Use https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/sect-using_chrony[`chronyd`] rather than `ntpd` or `systemd-timesyncd`.
+* `etcd` is not included in FCOS. Refer to xref:running-containers.adoc[Running etcd] for instructions to run it as a container on FCOS.
+* flannel is not supported. `fcct` should support the configuration you need without the use of flannel.
+* The functionality of the reboot manager (`locksmith`) is rolled into https://github.com/coreos/zincati/blob/master/README.md[`zincati`].
+* Along with Fedora 31, FCOS does not allow SSH password logins by default. For full details on how to set up users, refer to the documentation for the xref:ign-passwd.adoc[`passwd`] node of the FCC file.
+//* docker version selection
+* Because `usermod` is not yet fully-functional on FCOS, there is a `docker` group in the `/etc/group` file. This is a stop-gap measure to facilitate a smooth transition to FCOS. The team is working on a more functional `usermod`, at which time the `docker` group will no longer be included by default.
+* There is no way to create directories below the `root` directory. Changes are restricted to `/etc` and `/var`. Refer to the documentation for the xref:ign-storage.adoc[`storage`] node of the FCC configuration for details about writing directories and files to FCOS.
+
+== Operator notes
+* CL had three release channels: Stable, Beta, and Alpha. FCOS maintains the Fedora standard of two release streams: test and stable. See the https://getfedora.org/coreos/download/[download page] for more details.
+* Automatic updates are coordinated by Zincati, as described in the https://github.com/coreos/zincati/blob/master/docs/usage/auto-updates.md[Zincati documentation]. Each update requires a system reboot, and https://github.com/coreos/rpm-ostree/blob/master/README.md[`rpm-ostree`] provides a rollback mechanism in the grub menu.
+* The `update-ssh-keys` tool is not shipped on FCOS. The preferred method is to use the https://www.ssh.com/ssh/authorized_keys/openssh/[`AuthorizedKeysFile`].
+//*SELinux policy differences. What are they?
```

Comment:
That's a deep subject.  Broadly, SELinux existed on CL but wasn't very functional and I don't think many people used it.  Confinement on FCOS should work the same as in Fedora generally.